cdap-docs/_common/common-build.sh

```diff
@@ -236,12 +225,6 @@ function make_zip() {
   zip -qr ${ZIP_DIR_NAME}.zip ${PROJECT_VERSION}/* --exclude .DS_Store
```

Comment:
Ahh, so we're distributing the zip, not just using it, internally. Got it. That's a good enough reason. ZIP as a format is far more widespread and accessible.